### DISCUSSION TITLE
How can I copy content from the bundle to the currently active rootfs using a slot hook?

### DISCUSSION THREAD

**Discussion by martin-petzold** [rauc/rauc#1578] [Q&A]
I want to copy a file from the bundle to the currently active rootfs. However, I can't find the correct path to access the file in the bundle. I tried the following without success:

```
cp bootx64.efi /boot/efi/EFI/${RAUC_SLOT_BOOTNAME}/BOOT
AND
cp ${RAUC_MOUNT_PREFIX}/bootx64.efi /boot/efi/EFI/${RAUC_SLOT_BOOTNAME}/BOOT
```

Slot hook `hooks` (script in the bundle):

```
#!/bin/bash -ex

case "$1" in
	slot-post-install)
		test "${RAUC_SLOT_CLASS}" = "rootfs" || exit 0

		if [ ! -d "/boot/efi" ]; then
			echo "ERROR: /boot/efi does not exist!"

			exit 1
		fi

		mkdir -p /boot/efi/EFI/${RAUC_SLOT_BOOTNAME}/BOOT

		cp bootx64.efi /boot/efi/EFI/${RAUC_SLOT_BOOTNAME}/BOOT
		;;
	*)
		exit 1
		;;
esac

exit 0
```

RAUC content directory (`<INPUTDIR>` when using `rauc bundle <INPUTDIR> <BUNDLENAME>`):

```
manifest.raucm
<IMAGE>.rootfs.img
bootx64.efi
hooks
```

**Comment by martin-petzold** [comment]
It is: `${RAUC_MOUNT_PREFIX}/bundle/bootx64.efi`

**Comment by ejoerns** [comment]
It's not well-documented for hooks (only for [handlers](https://rauc.readthedocs.io/en/latest/reference.html#custom-handlers-interface)), but `RAUC_BUNDLE_MOUNT_POINT` should be the env variable you are looking for.
